src/frontend.js

```diff
@@ -20,22 +27,30 @@ const displayTodos = (todos) => {
     return todayCont;
   };
 
+  ul.appendChild(today());
+
   const addTodo = () => {
     const newTodoCont = document.createElement('li');
     newTodoCont.id = 'newTodoCont';
-    newTodoCont.setAttribute('id', 'newtodo');
+    // newTodoCont.setAttribute('id', 'newtodo');
 
     const todoText = document.createElement('input');
     todoText.type = 'text';
     todoText.placeholder = 'Add to your list...';
-    todoText.className = 'todo-text';
+    todoText.id = 'todo-text';
+    todoText.addEventListener('keydown', (e) => {
+      if (e.key === 'Enter') {
+        createTodos(todoText.value);
+        ul.appendChild(todoItem(todos[todos.length - 1]));
 
-    newTodoCont.appendChild(todoText);
+        const clear = document.getElementById('clear');
+        ul.appendChild(clear);
```

Comment:
- [ ] Kindly add the `Clear all completed` functionality, which clears all completed todo items when clicked using the `Array.filter` method. Check [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md#:~:text=Implement%20a%20function%20for%20the%20%22Clear%20all%20completed%22%20button%20(use%20filter()%20method).).

Reply:
@kimothokamau, we still need to attend to this comment.

kindly add a function to clear all completed tasks as stated in the project requirement.
Note: Make use of `Array.filer()` method.

Project Requirement Page:
![image](https://user-images.githubusercontent.com/31897434/126763425-709c3844-074e-4688-a983-560b055b26ac.png)

Reply:
This has been resolved. Please check on this loom video that the 'Clear all completed' function is working as per the requirement. https://www.loom.com/share/85cad2d663ec474c94df14b19340b20d

Reply:
@kimothokamau, apologies 🙏  for the late response.

The files changes mislead me a little bit, I later saw you have the function and also you added `filter` method.